### PR TITLE
Feature 147

### LINKED ISSUE #147: As a developer, I would like to have a guide explaining how I add content to the MkDocs wiki
**Is your feature request related to a problem? Please describe.**
Following #146 the wiki has been moved to MkDocs hosted on GH Pages.

**Describe the solution you'd like**
I would like to have a guide explaining how MkDocs works, and how I can make changes.

**Describe alternatives you've considered**
none

**Additional context**
nothing

### PR DESCRIPTION
**Pull request by mathiasandresen**
Document how the new wiki works.
closes #147

### CODE REVIEW

**Review by mads-bo — COMMENTED** [review]
@aau-giraf/sw607f20 you have been assigned this. Here's your checklist. 
Please use the following checklist:

- [ ] Is there documented which BLoCs/Screens the current class is related to?
- [ ] Is the responsibility/functionality of the current class explained? Also regarding the classes it is related to?
- [ ] If any unusual decisions are introduced in the code are they reasoned about in the documentation?

**Review by mads-bo — COMMENTED** [review]
@aau-giraf/sw613f20 you have been assigned this. Here's your checklist. 
Please use the following checklist:

- [ ] Is there documented which BLoCs/Screens the current class is related to?
- [ ] Is the responsibility/functionality of the current class explained? Also regarding the classes it is related to?
- [ ] If any unusual decisions are introduced in the code are they reasoned about in the documentation?

File: docs/writing_wiki.md [hunk 1]
```diff
@@ -5,11 +5,86 @@ title: "Editing the Wiki"
...
+### Running locally
+
+The wiki can be run locally by doing the following:
+
+* Install python 3
+    * check that pip is using python3 with ``pip -V``
+    * if not, check ``pip3 -V``
+* ``pip install -r requirements.txt`` (if pip is using python3)
+    * else ``pip3 install -r requirements.txt``
+* ``mkdocs serve``
```
Comment (Lortemanden) [inline docs/writing_wiki.md]: Can there be added a little more words. Is the "pip3 install -r requirements.txt" command supposed to be done by the command window or in Python? Also what dores mkdocs serve mean?

Comment (mathiasandresen) [reply docs/writing_wiki.md]: > 
> 
> Can there be added a little more words. Is the "pip3 install -r requirements.txt" command supposed to be done by the command window or in Python? Also what dores mkdocs serve mean?


See newest commit (3b8d9e1) is this better? :)

**Review by Lortemanden — DISMISSED** [review]

File: docs/writing_wiki.md (hunk 1 above)
Comment (Badgie) [inline docs/writing_wiki.md]: It is not common practice to install modules directly in your Python installation. See [venv](https://docs.python.org/3/library/venv.html).
After installing python and before installing requirements, you should include a step for setting up a `venv`, e.g.

- Set up a virtual environment (steps assume that your current dir is project root)
  - run `python3 -m venv venv` 
  - source the virtual environment 
    - Linux: `source venv/bin/activate`
    - Windows: `C:\> <venv>\Scripts\activate.bat`

File: docs/writing_wiki.md [hunk 2]
```diff
@@ -5,11 +5,94 @@ title: "Editing the Wiki"
...
+The wiki can be run locally by doing the following in a terminal:
+
+1. Install Python 3.
+2. Check that pip is using python3 with ``pip -V``.
+    * if not, check ``pip3 -V``
+3. Set up a virtual environment (steps assume that your current dir is project root) 
+      1. run ``python3 -m venv venv``
+      2. source the virtual environment
+            * Linux:    ``source venv/bin/activate`` 
+            * Windows:  ``.\venv\Scripts\activate.bat``
```
Comment (mathiasandresen) [inline docs/writing_wiki.md:32]: @Badgie i added this to the guide in eaa2508.

**Review by Badgie — APPROVED** [review]

**Review by Lortemanden — APPROVED** [review]

**Pull request merged**